src/styles/app/_overrides.scss

```diff
@@ -11,6 +11,10 @@
   }
 }
 
+.pf-c-alert-group.qpc-alert-group.pf-m-toast {
+  width: fit-content;
+}
+
```

Comment:
required:
lets look at moving this over to its own file like `_toastNotifications.scss`. we've been debating if we should get rid of the `override.scss` and `patternflyoverride.scss` files... since technically it's all overrides =)

Reply:
Good idea! Updated 👍